configure

```diff
@@ -617,6 +637,31 @@ if test $with_msan -eq 1; then
     echo >> configure.log
 fi
 
+if test $with_ubsan -eq 1; then
+    echo -n "Checking for undefined behavior sanitizer... " | tee -a configure.log
+    sanitizers=""
+    for san in  array-bounds bool bounds builtin enum float-cast-overflow float-divide-by-zero function integer-divide-by-zero local-bounds null nonnull-attribute object-size pointer-overflow return returns-nonnull-attribute shift shift-base shift-exponent signed-integer-overflow undefined unsigned-integer-overflow unsigned-shift-base vla-bound vptr; do
+        if try $CC -c $CFLAGS $test.c -fsanitize=$san ; then
+            if test -n "$sanitizers"; then
```

Comment:
Inconsistent style... Should be space on both sides of `;`.

Reply:
Fixed.